examples/sample-app/README.md

```diff
@@ -11,18 +11,52 @@ https://github.com/openshift/origin/blob/master/examples/sample-app/container-se
 
 Setup
 -----
-Before running these steps, you'll need to configure the docker daemon on your host to trust the docker registry service you'll be starting.
+At this stage of OpenShift 3 development, there are a few things that you will need to configure on the host where OpenShift is running in order for things to work.
+
+- - -
+
+**NOTE:** You do not need to do this is you are using [Vagrant](https://vagrantup.com/) to work with OpenShift (see the [Vagrantfile](https://github.com/openshift/origin/blob/master/Vagrantfile) for more info). These changes are only necessary when you have set up the host system yourself.
+
+- - -
+
+### Docker Changes ###
+
+First, you'll need to configure the docker daemon on your host to trust the docker registry service you'll be starting.
 
 To do this, you need to add "--insecure-registry 172.30.17.0/24" to the docker daemon invocation, eg:
-        
-        $ docker -d --insecure-registry 172.30.17.0/24
 
-If you are running docker as a service via systemd, you can add this argument to the options value in `/etc/sysconfig/docker`
+    $ docker -d --insecure-registry 172.30.17.0/24
+
+If you are running docker as a service via `systemd`, you can add this argument to the options value in `/etc/sysconfig/docker`
 
 This will instruct the docker daemon to trust any docker registry on the 172.30.17.0/24 subnet, rather than requiring a certificate.
 
 These instructions assume you have not changed the kubernetes/openshift service subnet configuration from the default value of 172.30.17.0/24.
 
+### SELinux Changes ###
+
+Presently the OpenShift 3 policies for SELinux are a work in progress. For the time being, to play around with the OpenShift system, it is easiest to downgrade the SELinux security level to 'permissive'. To do this, modify /etc/sysconfig/selinux as root and change the security level:
+
+    $ sudo vi /etc/sysconfig/selinux
+
+And change:
+
+    SELINUX=enforcing
+
+To:
+
+    SELINUX=permissive
+
+### FirewallD Changes ###
+
+Similar to our work on SELinux policies, the OpenShift firewalld rules are also a work in progress. For now it is easiest to disable firewalld altogether:
+
+    $ sudo systemctl stop firewalld
+    $ sudo systemctl disable firewalld
+
+### Reboot and Run ###
+
+Reboot your system and you will be ready to play with this OpenShift sample application.
```

Comment:
i'm worried we're going to lose a lot of users at "reboot your system".  I hate rebooting and i'd never do it just to try something out unless it was incredibly important to me.

i'm not aware of anyone else having had their system hang, so i think we might want to gather a little more data on why it failed for you before breaking out that hammer.